src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1193,6 +1194,37 @@ def _configure_security_policy_storage_params(arg_ctx):
                    arg_type=kid_param_type,
                    required=True,)
 
+    #####
+    #           sql managed instance ad-admin
+    ######
+    with self.argument_context('sql mi ad-admin') as c:
+        c.argument('managed_instance_name',
+                   arg_type=managed_instance_param_type)
+
+        c.argument('login',
+                   options_list=['--display-name', '-u'],
+                   help='Display name of the Azure AD administrator user or group.')
+
+        c.argument('sid',
+                   options_list=['--object-id', '-i'],
+                   help='The unique ID of the Azure AD administrator ')
+
+    with self.argument_context('sql mi ad-admin create') as c:
+        # Create args that will be used to build up the ManagedInstanceAdministrator object
+        create_args_for_complex_type(
```

Comment:
I think you can put this in `sql mi ad-admin` section so that it applies to both create and update (to avoid repetition)

Reply:
I've tried to do this, but when executing get or delete command get the following error: **ValueError: command authoring error: extra argument 'login' cannot be registered to a group-level scope 'sql mi ad-admin'. It must be registered to a specific command.**
So, I need to keep both "sql mi ad-admin create" and "sql mi ad-admin update" sections.